src/main/org/codehaus/groovy/util/ManagedReference.java

```diff
@@ -46,7 +46,6 @@ public final T get() {
     
     public final void clear() {
         ref.clear();
-        manager.removeStallEntries();
```

Comment:
Thanks for reviewing that other PR.  I merged that PR and will rebase this and leave the call to manager.removeStallEntries().